lib/view/home_page.dart

```diff
@@ -84,6 +84,7 @@ class _HomePageState extends State<HomePage> {
     final percentage = scrollPos < appBarBottom
         ? ((scrollPos) / appBarBottom.toDouble())
         : 1.00;
+    final elevation = percentage < 1.00 ? 0.00 : 4.00;
```

Comment:
According to the docs
> If this property is null, then [ThemeData.appBarTheme.elevation] is used, if that is also null, the default value is 4, the appropriate elevation for app bars.

It would be nice to be able to reference `ThemeData.appBarTheme.elevation` rather than hard coding to 4, but I have no idea how to do that 🤷 

Perhaps one of the more experienced flutter devs could suggest a change

Reply:
Would FlexibleSpaceBar work?